conans/client/cmd/export.py

```diff
@@ -96,6 +93,15 @@ def _capture_export_scm_data(conanfile, conanfile_dir, destination_folder, outpu
 
     _replace_scm_data_in_conanfile(os.path.join(destination_folder, "conanfile.py"), scm_data)
 
+    if captured:
+        # Generate the scm_folder.txt file pointing to the src_path
+        src_path = scm.get_repo_root()
+        scm_url = SCM.clean_url(scm_data.url)  # Remove peg_revision (SVN), ....
+        url_root = SCM(scm_data, src_path, output).get_remote_url(remove_credentials=True)
```

Comment:
This change is difficult to understand. I am getting the current folder pointing to a local path in disk. It is a bit confusing why it is necessary to ``get_remote_url()``. I understand in SVN it might make sense, but for git is very confusing. How can a ``os.path.relpath`` be computed from a scm_url and a url_root? 

Maybe this is some abstraction that should live in the SVN class, and have the Git one return empty or something?